README.md

```diff
@@ -1,12 +1,12 @@
-# ownCloud-Phoenix
+# ownCloud-Web
 
 Pushing a new design and frontend concept to ownCloud
 
 ![image](https://user-images.githubusercontent.com/25989331/63966638-fd4e0080-ca9b-11e9-931a-8dd9bf3ba82f.png)
 
-[See this online Demo](https://phoenix.owncloud.com) (user: demo, password: demo)
+[See this online Demo](https://web.owncloud.com) (user: demo, password: demo)
```

Comment:
And the online demo site has to change...

Reply:
As we're going to release the ownCloud Web as an oc10 app, maybe we could drop this instance and install it as an app on demo.owncloud.com. The Phoenix demo instance was anyway already using it as a server so the only difference would be `https://demo.owncloud.com/apps/web` instead of `https://phoenix.owncloud.com`